examples/hosts/app-integration/external-data/tests/mockService.test.ts

```diff
@@ -52,16 +53,24 @@ describe("mockCustomerService", () => {
     /* eslint-disable @typescript-eslint/no-unsafe-call, @typescript-eslint/no-unsafe-member-access */
 
     it("fetch-tasks: Ensure server yields the data we expect", async () => {
-        const expectedData = await externalDataSource!.fetchData();
-        await request(server!).get("/fetch-tasks").expect(200, {taskList: expectedData});
+        const expectedData = await externalDataSource!.fetchData().then((data)=>{
```

Comment:
Can't we just await the data here, rather than calling `.then`? This test is async.